apps/judicial-system/web/src/routes/Admin/Statistics/components/StatisticsCSVButton.tsx

```
@@ -0,0 +1,152 @@
+import { Button } from '@island.is/island-ui/core'
+import { formatDate } from '@island.is/judicial-system/formatters'
+import { CaseStatistics } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { mapServiceStatusTitle } from '../helpers'
+
+const convertToCSV = (rows: Record<string, unknown>[]): string => {
+  if (!rows.length) return ''
+
+  const headers = Array.from(new Set(rows.flatMap((row) => Object.keys(row))))
+
+  const escape = (val: unknown): string => {
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)